modules/network/network_test.go

```diff
@@ -115,7 +115,7 @@ func TestCreateNetwork(t *testing.T) {
 
 	storage := filepath.Join(dir, netName)
 	networker := &networker{
-		nodeID:     node1.NodeID,
+		//nodeID:     node1.NodeID,
```

Comment:
Left over ?

Reply:
Yeah, This caused some tests to fail. and I was testing something, I will spend sometimes on getting these to work properly.